lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -10449,18 +10459,30 @@ namespace Js
     {
         Var func = JavascriptOperators::GetProperty(iterator, PropertyIds::next, scriptContext);
 
+        ThreadContext *threadContext = scriptContext->GetThreadContext();
         if (!JavascriptConversion::IsCallable(func))
         {
+            if (!threadContext->RecordImplicitException())
+            {
+                return scriptContext->GetLibrary()->GetUndefined();
+            }
             JavascriptError::ThrowTypeError(scriptContext, JSERR_NeedFunction);
         }
 
         RecyclableObject* callable = RecyclableObject::FromVar(func);
         Js::Var args[] = { iterator, value };
         Js::CallInfo callInfo(Js::CallFlags_Value, _countof(args) + (value == nullptr ? -1 : 0));
-        Var result = JavascriptFunction::CallFunction<true>(callable, callable->GetEntryPoint(), Js::Arguments(callInfo, args));
+        Var result = threadContext->ExecuteImplicitCall(callable, ImplicitCall_Accessor, [&]() -> Var
+            {
```

Comment:
Also in this file - I have seen few other places we have ExecuteImplicitCall, but the whole function is wrapped under ExecuteImplicitCall, as opposed to what you have done (just the callfunction). Was that oversight?